Dockerfile

```diff
@@ -10,12 +10,13 @@ RUN apt update && apt install -y protobuf-compiler
 RUN go install google.golang.org/protobuf/cmd/protoc-gen-go \
     google.golang.org/grpc/cmd/protoc-gen-go-grpc \
     github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
-    github.com/googleapis/gnostic/apps/protoc-gen-openapi
+    github.com/google/gnostic/cmd/protoc-gen-openapi
 
 ADD . .
 
 RUN go generate ./...
-RUN go build ./...
+RUN go build -o /build/engine cmd/engine/engine.go
+RUN go build -o /build/cl cmd/cli/cl.go
```

Comment:
Alternatively, we could do `RUN go build -o /build/ ./...`. This comes closer to the previous version (`RUN go build ./...`)